src/styles/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  font-size: 12px;
+  font-weight: 400;
+  color: $main-color;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: block;
+    align-self: center;
+    margin-bottom: 40px;
+    width: 160px;
+  }
+
+  &__title {
+    font-size: $size;
+    line-height: 18px;
+    margin-bottom: 4px;
+    font-weight: 500;
+  }
+
+  &__label {
+    display: block;
+    font-size: $min-size;
+    line-height: $min-line-height;
+
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__review-label {
+    display: block;
+
+    margin-left: 17px;
```

Comment:
u don't need this margin.
use `justify-content: space-between` property in `card__review` class instead